DiscordBeam/Base/src/main/java/de/lukweb/discordbeam/FileExtensionMarkdownLanguage.java

```diff
@@ -0,0 +1,27 @@
+package de.lukweb.discordbeam;
+
+public enum FileExtensionMarkdownLanguage {
+	KOTLIN("kt", "kotlin");
+
+	private final String extension;
+	private final String language;
+
+	FileExtensionMarkdownLanguage(String extension, String language) {
+		this.extension = extension;
+		this.language = language;
+	}
+
+	public static FileExtensionMarkdownLanguage getByExtension(String extension) {
+		for (FileExtensionMarkdownLanguage fileExtensionMarkdownLanguage : FileExtensionMarkdownLanguage.values()) {
+			if (fileExtensionMarkdownLanguage.extension.equals(extension)) {
+				return fileExtensionMarkdownLanguage;
+			}
+		}
+
+		return null;
+	}
+
+	public String getLanguage() {
+		return language;
+	}
+}
```

Comment:
It would be nice, if there was a new line at the end of the file.

Reply:
Will do 👍